firebase-vscode/src/core/index.ts

```diff
@@ -39,6 +40,14 @@ export async function registerCore(
     vscode.env.openExternal(vscode.Uri.parse(href));
   });
 
+  const executeCommand = setupExecuteCommandMockable(context);
+
+  context.subscriptions.push({
```

Comment:
assign to value and add to Disposable at bottom of file

Reply:
Why is this preferable?

Reply:
Instead of writing docs, just having the named variable could really help readability :)

Reply:
What doc? :)

This is IMO more maintainable to the old way of returning a Disposable.
It makes it much more obvious what's disposed and what isn't. And it makes it much simpler to guarantee that the disposal happens in the correct order too.

In the grand scheme of things, is it really something that should block this PR?
If you insist, I can change it. But I really think this is a better way of disposing objects.

Reply:
Docs as in comments. It's less important in this PR, but the larger config PR has >10 + subscriptions that I have to read in detail to understand what's going on. Adding a name to them, and returning the disposables together will give me a quick look at all the subscriptions in a file/method. 

If this is truly something you want to keep, then I'll ask for a comment describing each "context.subscriptions.push" block.

Reply:
Do you have an example of what that comment would be?
I'm a bit at loss at what you're asking here.